BPCompatibleAlertController/BPCompatibleAlertController.swift

```diff
@@ -31,9 +51,24 @@ public class BPCompatibleAlertController : NSObject, UIAlertViewDelegate {
     */
     public var alertViewStyle: UIAlertViewStyle
     
+    /**
+    If this set, when the user dismisses the alert all the resources used
+    by this alert controller will be automatically released allowing this object to be deallocated.
+    
+    After the user has dismissed the alert, presentFrom: cannot be called again on this object.
+    
+    Should client code set this to false (because they intend to re-user the alert), it must at some stage call releaseResources() to
+    ensure internal resources are freed up.
+    
+    releaseResourcesWhenAlertDismissed defaults to true.
+    */
+    public var releaseResourcesWhenAlertDismissed: Bool = true
+    
     private var alertController: UIAlertController!
-    private var alertView: UIAlertView!
+    private var alertView: BPAlertView!
     private var actions: [String : BPCompatibleAlertAction]
+    private var actionObservers: Array<NSObjectProtocol> = []
```

Comment:
Sure, consistency is good. You mean to something like this `private var actionObservers: [NSObjectProtocol] = []`? Let me know and I'll update the pull request.